ca/ca.go

```diff
@@ -29,7 +29,7 @@ var (
 )
 
 type NewOpts struct {
-	IssuerId   string
+	Issuer     mtc.OID
```

Comment:
This is not an OID, as OIDs are encoded differently. Instead it's an *arc under the PEN OID*, but that doesn't really roll off the tongue either.

Reply:
Both 1234.1.2 and 1234.1 can be trust anchor identifiers in the more general context of [TLS Trust Anchor Identifiers](https://www.ietf.org/archive/id/draft-beck-tls-trust-anchor-ids-00.html) so we don't really need the two different types. If you keep it split, then I'd rename `OID` to `RelativeOID`, `PENArc` or something similar.